src/config.ts

```diff
@@ -315,7 +317,16 @@ export const getAllowedAccountListClientIDs: string[] = [
   RURAL_PAYMENT_WALES_NON_PROD,
 ];
 
-export const hmrcClientIds: string[] = [HMRC_NON_PROD, "hmrc"];
```

Comment:
i think we don't need to recreate the array here...

In this file, for example: https://github.com/govuk-one-login/di-account-management-frontend/blob/main/src/utils/yourServices.ts#L100

We could have something like...

```
const hasDetailedCard = getClient(getAppEnv(), service.client_id).isHmrc
```